src/compas_timber/ghpython/components/CT_Bake_BoxMap/code.py

```diff
@@ -78,7 +78,31 @@ def create_box_map(pln, sx, sy, sz):
         dx = Interval(-sx * 0.5, sx * 0.5)
         dy = Interval(-sy * 0.5, sy * 0.5)
         dz = Interval(-sz * 0.5, sz * 0.5)
+        if swap_uv:
```

Comment:
so is it ok that there's only one `swap_uv`? this will flip the coordinates for both beams and plates right? can it be user might want to flip for just one of them?